### PR TITLE
[MINOR] fix logger number not correct

### PR DESCRIPTION
**Pull request by Shoothzj**
### Motivation

(Explain: why you're making that change, what is the problem you're trying to solve)

### Changes

fix logger number not correct

### CODE REVIEW

**Comment by Shoothzj** [comment]
rerun failure checks

**Comment by Shoothzj** [comment]
rerun failure checks

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/datainteg/DataIntegrityCookieValidation.java [hunk 1]
```diff
@@ -96,7 +96,7 @@ private void stampCookie(Cookie masterCookie, Version expectedVersion, List<File
                 log.info("Stamping cookie to directory {}", d);
                 masterCookie.writeToDirectory(d);
             } catch (IOException ioe) {
-                log.error("Exception writing cookie to {}", ioe);
+                log.error("Exception writing cookie to ", ioe);
```
Comment (nicoloboschi) [inline bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/datainteg/DataIntegrityCookieValidation.java:99]:
```suggestion
                log.error("Exception writing cookie", ioe);
```

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/AuthHandler.java [hunk 2]
```diff
@@ -151,7 +151,7 @@ && checkAuthPlugin(req.getAuthRequest(), ctx.channel())) {
         private boolean checkAuthPlugin(AuthMessage am, final Channel src) {
             if (!am.hasAuthPluginName() || !am.getAuthPluginName().equals(authProviderFactory.getPluginName())) {
                 LOG.error("Received message from incompatible auth plugin. Local = {}," + " Remote = {}, Channel = {}",
```
Comment (nicoloboschi) [inline bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/AuthHandler.java:153]: can we remove the concatenation ?

**Review by nicoloboschi — CHANGES_REQUESTED** [review]

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/AuthHandler.java (hunk 2 above)
Comment (Shoothzj) [reply bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/AuthHandler.java]: fixed

**Comment by Shoothzj** [comment]
rerun failure checks

**Comment by Shoothzj** [comment]
rerun failure checks

**Review by nicoloboschi — APPROVED** [review]

**Comment by Shoothzj** [comment]
rerun failure checks

**Comment by Shoothzj** [comment]
rerun failure checks

**Comment by Shoothzj** [comment]
rerun failure checks

**Review by zymap — APPROVED** [review]

**Pull request merged**